frontend/src/scenes/instance/SystemStatus/RenderMetricValue.tsx

```diff
@@ -3,20 +3,34 @@ import { humanFriendlyDetailedTime } from 'lib/utils'
 import React from 'react'
 import { InstanceSetting } from '~/types'
 import { MetricRow } from './systemStatusLogic'
+import { LockOutlined } from '@ant-design/icons'
 
 const TIMESTAMP_VALUES = new Set(['last_event_ingested_timestamp'])
 
 type BaseValueInterface = Pick<MetricRow, 'key' | 'value'> & Partial<Pick<InstanceSetting, 'value_type'>>
 export interface MetricValueInterface extends BaseValueInterface {
     emptyNullLabel?: string
+    is_secret?: boolean
 }
 
 export function RenderMetricValue({
     key,
     value,
     value_type,
     emptyNullLabel,
+    is_secret,
 }: MetricValueInterface): JSX.Element | string {
+    if (is_secret) {
+        return (
+            <LemonTag style={{ color: 'var(--text-muted)', padding: '4px 6px', backgroundColor: '#fee5b3' }}>
+                <div style={{ display: 'inline-flex', alignItems: 'center' }}>
+                    {is_secret && <LockOutlined style={{ marginRight: 4, color: 'var(--warning)' }} />}
```

Comment:
Actually it'd be great if `LemonTag` had an `icon` prop, as this is looks kind of messy one-off, while it's super likely we'll reuse this pattern. Also, let's use Material icons instead (in this case, our `IconLock` instead of Ant's `LockOutlined`).

Reply:
Great idea! Will do